File: tests/hwk-tester/leap-year/spec/leap-year-spec.js
var leapYearCalculator = require("../leap-year");

describe("isLeapYear", function () {
  it("should return false for the year 1901", function () {
    var result = leapYearCalculator.isLeapYear(1901);
    expect(result).toBe(false);
  });
  it("should return true for the year 1904", function () {
    var result = leapYearCalculator.isLeapYear(1904);
    expect(result).toBe(true);
  });
  it("should return false for the year 1900", function () {
    var result = leapYearCalculator.isLeapYear(1900);
    expect(result).toBe(false);
  });
  it("should return true for the year 2000", function () {
    var result = leapYearCalculator.isLeapYear(2000);
    expect(result).toBe(true);
  });
});

File: tests/hwk-tester/leap-year/spec/leap_year_spec.rb
require_relative '../leap_year'

describe 'leap_year?' do
  it "should return false for the year 1900" do
    expect(leap_year?(1900)).to eq(false)
  end

  it "should return false for the year 1901" do
    expect(leap_year?(1901)).to eq(false)
  end

  it "should return true for the year 1904" do
    expect(leap_year?(1904)).to eq(true)
  end

  it "should return true for the year 2000" do
    expect(leap_year?(2000)).to eq(true)
  end
end
